kamu-adapter-graphql/src/queries/metadata_chain.rs

```diff
@@ -77,6 +78,29 @@ impl MetadataChain {
         Ok(block.map(|b| MetadataBlockExtended::new(hash, b, Account::mock())))
     }
 
+    /// Returns a metadata block corresponding to the specified hash and encoded in desired format
+    async fn block_by_hash_raw(
```

Comment:
I suggest using this method for the YAML view - this is the only way we will display ALL fields of the metadata block, unlike when converting GQL object to YAML, since latter will only contain fields that platform is aware of (i.e. queries via GQL).

Reply:
Why calling it "raw"? It looks like "export as" to me.

Reply:
"Raw" as in returning a serialized data as opposed to returning a GQL object. But I agree it's not a great name.

Renamed to `block_by_hash_encoded` as "export" didn't sound right to me and I also wanted this method to have the same name prefix as the one returning a GQL object.